src/thin_client.rs

```diff
@@ -177,9 +177,12 @@ impl ThinClient {
             }
             self.process_response(&resp);
         }
+        trace!("get_balance {:?}", self.balances.get(pubkey));
+        //TODO: call the contract specific get_balance for contract_id's thin client can introspect
```

Comment:
I don't understand this comment. Missing words?

Reply:
Ah, I see that this parallels this comment: https://github.com/solana-labs/solana/blob/884ebc69c813963b6f79ee3cc42e128ffc8b002b/src/bank.rs#L331
But it could use some editing to stand alone and be grammatical.